content/behind_the_scenes/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Behind The Scenes"
+menu:
+  main:
+    weight: 1
+showDate: false
+draft: false
+---
+
+## Overview ##
+
+This article aims to explain how we have re-designed and re-written the backend used to collect, enrich and display the ADSB data our community sends us. 
+
+## Anatomy of a Feeder Connection ##
+When you follow our [docker_planewatch] 
+![plane.watch feeder connection anatomy](Connection%20Anatomy.drawio.png)
+
+1. [pw-feeder][pw-feeder] opens two connections to [Bordercontrol](#bordercontrol) - one connection for [BEAST][beast protocol], and another for [MLAT][mlat-client].
+2. During connection establishment, [Bordercontrol](#bordercontrol) checks the validity of the feeder's API key. If invalid, the connection is dropped. If valid, the connection is allowed, and [bordercontrol](#bordercontrol) starts a "feed-in" container for the client.
+3. Bordercontrol proxies [BEAST][beast protocol] traffic to the feed-in container.
+4. Bordercontrol proxies [MLAT][mlat-client] traffic to an [mlat_server][mlat-server] instance running on the regional multiplexer.
+5. For the "new" environment, [pw_ingest](#pw_ingest) running within the feed-in container decodes the [BEAST][beast protocol] data, and publishes the data as a message onto the NATS message bus. The data is processed through the [pw-pipeline][pw-pipeline].
+6. For the "legacy" environment, the [BEAST][beast protocol] data for each region is multiplexed in each regional multiplexer. Virtual Radar Server then consumes this data for the legacy front-end.
+
+## Components ##
+
+### ADS-B Traffic Control (ATC) ###
+
+[ATC][atc] is [plane.watch][plane.watch]'s feeder portal. It provides:
+
+* User sign-up and account management
+* Feeder creation and management
+* Databases containing aircraft types, airports, routes, operators and waypoints
+
+[ATC][atc] provides both public and private APIs. The public API is used by [pw-feeder](#pw-feeder) to report the feeder's status back to the client.
+
+The private API is used to:
+
+* Authenticate feeders upon connection to [bordercontrol](#bordercontrol)
+* Provide aircraft types, airports, routes, operators and waypoints to the [Enrichment Centre](#pw-enrichment)
+
+### pw-feeder ###
+
+[pw-feeder][pw-feeder], simply put, it is a [BEAST][beast protocol] and [MLAT][mlat] protocol specific [stunnel][stunnel] client, that securely proxies data from the client to [plane.watch][plane.watch]'s servers (specifically [bordercontrol](#bordercontrol) - see below).
+
+[pw-feeder][pw-feeder] runs on the feeder client computer, and receives [BEAST protocol][beast protocol] data from software such as [dump1090][dump1090] and [readsb][readsb], or from hardware devices such as the [Jetvision Radarcape][radarcape]. It also communicates with [mlat-client][mlat-client] software running on the client, and mlat-server software running within [plane.watch][plane.watch].
```

Comment:
I'm nitpicking now but in the diagram you call it "feeder computer", not "feeder client computer".